src/ngraph/codegen/toolchains/Cuda.h

```diff
@@ -0,0 +1,203 @@
+//===--- Cuda.h - Cuda ToolChain Implementations ----------------*- C++ -*-===//
+//
```

Comment:
This is going to need to go through protex.
@aslepko

Reply:
We also don't want clang-format style applied to these files. Not sure how we do this.